modules/ob-creating-a-buildah-build.adoc

```diff
@@ -43,16 +46,33 @@ EOF
 <2> The build strategy that you use to build the container.
 <3> A name-value list to specify values for parameters defined in the build strategy. To set the value of the `dockerfile` strategy parameter, specify the Dockerfile location required to build the output image.
 <4> The location where the built image is pushed. In this procedural example, the built image is pushed to the {ocp-product-title} cluster internal registry. `buildah-example` is the name of the current project. Ensure that the specified project exists to allow the image push.
++
+.Using `shp` CLI
```

Comment:
Do the callouts from the previous example also apply to the `shp` CLI example? Maybe a way exsits to group the callouts?